src/panels/beaconSummaryPanel/BeaconSummaryEditing.test.tsx

```diff
@@ -98,4 +95,23 @@ describe("BeaconSummaryEditing", () => {
       expect(onCancel).toHaveBeenCalled();
     });
   });
+
+  it("does not allow user to edit derived fields", async () => {
+    render(
+      <BeaconSummaryEditing
+        beacon={beaconFixture}
+        onSave={jest.fn()}
+        onCancel={jest.fn()}
+      />
+    );
+
+    expect(
+      await screen.findByText(beaconFixture.protocolCode, { exact: false })
+    ).toBeVisible();
+    expect(
+      await screen.findByText(beaconFixture.codingMethod, { exact: false })
+    ).toBeVisible();
+    expect(screen.queryByDisplayValue(beaconFixture.protocolCode)).toBeNull();
+    expect(screen.queryByDisplayValue(beaconFixture.codingMethod)).toBeNull();
+  });
```

Comment:
"expect to see the text on the page, but don't expect to see a form input with the same display value"